frontend/package.json

```diff
@@ -0,0 +1,39 @@
+{
+  "name": "capstone",
+  "version": "0.1.0",
+  "private": true,
+  "dependencies": {
+    "@testing-library/jest-dom": "^5.16.5",
+    "@testing-library/react": "^13.4.0",
+    "@testing-library/user-event": "^13.5.0",
```

Comment:
It is advisable to include testing libraries in the devDependencies section, as failing to do so may result in their being bundled into the React application, consequently leading to an increase in the overall bundle size.

Reference -> https://www.geeksforgeeks.org/difference-between-dependencies-devdependencies-and-peerdependencies/